pkg/flowaggregator/flowaggregator.go

```diff
@@ -324,29 +350,20 @@ func (fa *flowAggregator) initExportingProcess() error {
 		return fmt.Errorf("got error when initializing IPFIX exporting process: %v", err)
 	}
 	fa.exportingProcess = ep
-	// Currently, we send two templates for IPv4 and IPv6 regardless of the IP families supported by cluster
-	fa.templateIDv4 = fa.exportingProcess.NewTemplateID()
-	bytesSent, err := fa.sendTemplateSet(false)
-	if err != nil {
-		fa.exportingProcess.CloseConnToCollector()
-		fa.exportingProcess = nil
-		fa.templateIDv4 = 0
-		fa.set.ResetSet()
-		return fmt.Errorf("sending IPv4 template set failed, err: %v", err)
+	// Currently, we send 4 templates for covering the cases in dual stack where Pod IPs and
```

Comment:
"Currently, we send 4 templates for covering all the cases in dual-stack clusters, where Pod IPs and original exporter IP could belong to different IP families."